src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        ColorSupplier supplier = new ColorSupplier();
+        Random random = new Random();
+        return new Ball(supplier.getRandomColor(), random.nextInt(100));
```

Comment:
All magic numbers in your code should be constants. How do you think what max number will be returned by random.nextInt(100);?

Reply:
I got your point, a random integer between 0 (inclusive) and 100 (exclusive).. I'll remember it next time.